src/auth/screens/events.screen.js

```diff
@@ -29,52 +30,58 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
-const styles = StyleSheet.create({
-  descriptionContainer: {
-    justifyContent: 'center',
-    flex: 1,
-    marginLeft: 10,
-    color: colors.primaryDark,
-    ...fonts.fontPrimaryLight,
-  },
-  linkDescription: {
-    ...fonts.fontPrimarySemiBold,
-  },
-  linkBranchDescription: {
-    ...fonts.fontCode,
-  },
-  deletedLinkBranchDescription: {
-    color: colors.greyDarkest,
-    ...fonts.fontCode,
-  },
-  date: {
-    color: colors.greyDark,
-  },
-  subtitleContainer: {
-    paddingHorizontal: 15,
-    paddingBottom: 10,
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  subtitle: {
-    color: colors.greyDark,
-    fontSize: normalize(11),
-    marginTop: 1,
-    ...fonts.fontPrimarySemiBold,
-    marginLeft: 39,
-  },
-  textContainer: {
-    flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
-    marginHorizontal: 15,
-  },
-  noneTitle: {
-    fontSize: normalize(14),
-    textAlign: 'center',
-    ...fonts.fontPrimary,
-  },
-});
+const DescriptionContainer = styled.Text`
+  justify-content: center;
+  flex: 1;
+  margin-left: 10;
+  color: ${colors.primaryDark};
+  font-family: ${fonts.fontPrimaryLight.fontFamily};
+`;
+
+const LinkDescription = styled.Text`
+  font-family: ${fonts.fontPrimarySemiBold.fontFamily};
+`;
+
+const LinkBranchDescription = styled.Text`
+  font-family: ${fonts.fontCode.fontFamily};
+`;
+
+const DeletedLinkBranchDescription = styled.Text`
+  color: ${colors.greyDarkest};
+  font-family: ${fonts.fontCode.fontFamily};
+`;
+
+const Date = styled.Text`
```

Comment:
"Date" sounds too general. It usually reminds me "new Date()" in Javascript. How about "DateText" or something else?